element_moseq/moseq_infer.py

```diff
@@ -447,24 +469,8 @@ def make_insert(
             }
         )
 
-        # Insert motion sequence results
-        for result_idx, result in results.items():
-            self.MotionSequence.insert1(
-                {
-                    **key,
-                    "video_name": result_idx,
-                    "syllable": result["syllable"],
-                    "latent_state": result["latent_state"],
-                    "centroid": result["centroid"],
-                    "heading": result["heading"],
-                    "file_csv": (
-                        inference_output_dir / "results_as_csv" / f"{result_idx}.csv"
-                    ).as_posix(),
-                }
-            )
+        # Add key to each motion sequence record and insert
+        self.MotionSequence.insert(motion_sequence_data)
```

Comment:
It may be more advantageous to do single insert1 in a loop to maintain the connection to the db (prevent timeout error), especially when inserting files (could be slow).
This is a general suggestion, it's probably fine here